src/plugins/welcome/introductionwidget.cpp

```diff
@@ -120,9 +122,9 @@ IntroductionWidget::IntroductionWidget(QWidget *parent)
             "<tr><td>Debug:</td><td>Analyze your application with a debugger or other "
             "analyzers.</td></tr>"
             "<tr><td>Projects:</td><td>Manage project settings.</td></tr>"
-            "<tr><td>Sailfish OS:</td><td>Control the Sailfish SDK.</td></tr>"
+            "<tr><td>%1:</td><td>Control the %2.</td></tr>"
             "<tr><td>Help:</td><td>Browse the help database.</td></tr>"
-            "</table></p>")},
+            "</table></p>").arg(Sfdk::Sdk::osVariant()).arg(Sfdk::Sdk::sdkVariant())},
```

Comment:
Control centre embedding will be removed soon https://github.com/sailfishos/sailfish-qtcreator/pull/228

Reply:
Undo changes with control centre